.github/mergify.yml

```diff
@@ -0,0 +1,12 @@
+queue_rules:
+  - name: default
+    batch_size: 8
+    checks_timeout: 10800 s
+    merge_method: squash
+    queue_conditions:
+      - "#approved-reviews-by >= 1"
```

Comment:
Happy to leave this in, although it could also be nice to minimise the number of rules in the config when they can be specified elsewhere (more flexibility). We can roll with it for a while and tweak as necessary.